src/plugins/teleport-plugin-vue-base-component/utils.ts

```diff
@@ -28,7 +28,15 @@ export const generateVueNodesTree = (
 
   if (attrs) {
     Object.keys(attrs).forEach((attrKey) => {
-      addAttributeToNode(htmlNode, name, attrKey, attrs[attrKey], dataObject)
+      const attrValue = attrs[attrKey]
+      // arrays are moved to the data object and referenced as dynamic keys (binding)
+      if (attrValue.type === 'static' && Array.isArray(attrValue.content)) {
```

Comment:
same as before, maybe we should switch or something and throw if type not recognized.